src/rtc.c

```diff
@@ -810,16 +810,8 @@ void RTC_StartAlarm(alarm_t name, uint8_t day, uint8_t hours, uint8_t minutes, u
   */
 void RTC_StopAlarm(alarm_t name)
 {
-  /* Clear RTC Alarm Flag */
-#ifdef RTC_ALARM_B
-  if (name == ALARM_B) {
-    __HAL_RTC_ALARM_CLEAR_FLAG(&RtcHandle, RTC_FLAG_ALRBF);
-  } else
-#endif
-  {
-    __HAL_RTC_ALARM_CLEAR_FLAG(&RtcHandle, RTC_FLAG_ALRAF);
-  }
-  /* Disable the Alarm A interrupt */
+  /* Clear RTC Alarm Flag : done in the HAL_RTC_DeactivateAlarm function */
+  /* Disable the Alarm  interrupt */
```

Comment:
Seems not the case for all series. So it should be fine to keep this

Reply:
restored